src/features/onboarding/passwordRecovery/PasswordRecovery.tsx

```diff
@@ -1,44 +1,108 @@
-import * as React from 'react'
-import { useHistory } from 'react-router-dom'
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
 import QRCode from 'qrcode.react'
-
+import MultiToggleSwitch from '../../../common/components/MultiToggleSwitch'
 import Typography from '../../../common/components/Typography/Typography'
-import { colors } from '../../../common/theme/colors'
 import * as ROUTES from '../../../common/utils/constants/routes'
-
-import * as Styles from './PasswordRecovery.styles'
+import Tooltip from '../../../common/components/Tooltip'
+import { Button } from '../../../common/components/Buttons'
+import infoIco from '../../../common/assets/icons/ico-info.svg'
 
 const PasswordRecovery: React.FC = () => {
-  const history = useHistory()
-
+  const [tab, setTab] = useState(0)
```

Comment:
Non-blocking : consider using enum instead of number, like:
```
enum Tab {
  qrCode,
  keys
}
...
const [tab, setTab] = useState(Tab.qrCode)
...
{tab === Tab.qrCode && (..)}
```

Reply:
fixed